tests/components/axis/test_light.py

```diff
@@ -113,8 +123,8 @@ async def test_lights(hass):
     device.api.event.process_event(EVENT_OFF)
     await hass.async_block_till_done()
 
-    light_0 = hass.states.get(f"light.{NAME}_ir_light_0")
-    assert light_0.state == "off"
+    light_0 = hass.states.get(entity_id)
+    assert light_0.state == STATE_OFF
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)